#id number
int32 id

#time created
time time

#pose
geometry_msgs/Pose pose

#robot id assigned
int32 robotID